mozilla-release/mobile/android/base/java/org/mozilla/gecko/controlcenter/Categories.java

```diff
@@ -10,15 +10,15 @@
  */
 public enum Categories {
 
-    advertising(R.color.advertising, R.drawable.ic_category_advertising),
-    audio_video_player(R.color.audio_video_player, R.drawable.ic_category_audio_video),
-    comments(R.color.comments, R.drawable.ic_category_comments),
-    customer_interaction(R.color.customer_interaction, R.drawable.ic_category_customer_interaction),
-    essential(R.color.essential, R.drawable.ic_category_essential),
-    pornvertising(R.color.pornvertising, R.drawable.ic_category_pornvertising),
-    site_analytics(R.color.site_analytics, R.drawable.ic_category_site_analytics),
-    social_media(R.color.social_media, R.drawable.ic_category_social_media),
-    unknown(R.color.default_category, R.drawable.ic_category_pornvertising); //TODO get icon for unknown
+    advertising(R.color.advertising, R.color.advertising_disabled, R.drawable.ic_category_advertising),
```

Comment:
The drawable should be named `cc_ic_category_advertising`, all the stuff relative to the controll center should be prepended with `cc_` or `ghostery_`